docs/lib/storage/fragments/flutter/configureaccess.md

```diff
@@ -0,0 +1,92 @@
+ When adding the Storage category, you configure the level of access authenticated and guest users have to your S3 bucket. Additionally, when uploading files using the Storage category, you can specify the access level for that file to be either public, protected, or private.
+
+- **Public** Accessible by all users
+- **Protected** Readable by all users, but only writable by the creating user
+- **Private** Readable and writable only by the creating user
+
+For protected and private access, the `user_id` in the prefix corresponds to the unique ID for the creating user.
+
+<amplify-callout>
+
+The default access level for the Storage category is **public**. Unless you specify otherwise, all uploaded files will be publicly available for all users.
+
+</amplify-callout>
+
+## Protected access
+
+Create an options object specifying the protected access level to allow other users to read the object:
+
+```dart
+try {
+  // use a file selection mechanism of your choice
+  File local = await FilePicker.getFile(type: FileType.image);
+  final key = new DateTime.now().toString();
+  final path = local.absolute.path;
+  S3UploadFileOptions options = S3UploadFileOptions(
+    accessLevel: StorageAccessLevel.protected
+  );
+  UploadFileResult result = await Amplify.Storage.uploadFile(
+    key: key,
+    path: path,
```

Comment:
This needs to  use `local` (File) instead of `path`(String) in Storage.uploadFile().

Reply:
Done.

Reply:
Done.